openstack/identity/v3/tokens/requests.go

```diff
@@ -182,7 +182,7 @@ func Get(c *gophercloud.ServiceClient, token string) (r GetResult) {
 func Validate(c *gophercloud.ServiceClient, token string) (bool, error) {
 	resp, err := c.Request("HEAD", tokenURL(c), &gophercloud.RequestOpts{
 		MoreHeaders: subjectTokenHeaders(c, token),
-		OkCodes:     []int{200, 204, 400, 401, 403, 404},
+		OkCodes:     []int{200, 204, 400, 403, 404},
```

Comment:
> Should those be removed, too?

Yes

> To confirm, an "OK Code" is one that would only signify a successful action? Not one that is necessarily used in the API service itself?

Correct. In fact, the only reason the `Validate` function accepts 404 is for a legacy reason. It returns a `bool`, and 404 there would signify `false` (i.e. unauthorized).